src/main/java/com/github/hiroyuki_sato/digdag/plugin/SshOperatorFactory.java

```diff
@@ -61,6 +64,9 @@ public SshOperator(OperatorContext context)
         }
 
         private final int defaultCommandTimeout = 60;
+        private final int initialRetryWait = 500;
+        private final int maxRetryWait = 2000;
+        private final int maxRetryLimit = 3;
 
```

Comment:
(1) What do you think to add `default_` prefix?
Like `defaultInitialRetryWait` or `DEFAULT_INITIAL_RETRY_WAIT`
Ref: https://github.com/treasure-data/digdag/pull/994/files#diff-84a9dc43263ac1f78fa3b2aca8234f6fR160

(2) `private static final` is better?

Reply:
(1)
Just I forgot default prefix.

(2)
yes, private static final is better, and I'll change defaultCommandTimeout, too.